commons/ava_services/gecko_service_initializer_core.go

```diff
@@ -1,17 +1,29 @@
 package ava_services
 
 import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
 	"fmt"
 	"github.com/docker/go-connections/nat"
 	"github.com/kurtosis-tech/kurtosis/commons/services"
+	"github.com/palantir/stacktrace"
 	"github.com/sirupsen/logrus"
+	"math/big"
+	"os"
 	"strconv"
 	"strings"
+	"time"
 )
 
 const (
 	httpPort = 9650
 	stakingPort = 9651
+	stakingTlsCertPath = "node.crt"
```

Comment:
Per the comment on the other PR, this should be the full path inside the Docker image to where the InitializerCore wants to mount these files

Reply:
done